src/plugins/bfetch/server/plugin.ts

```diff
@@ -110,7 +110,7 @@ export class BfetchServerPlugin
           'Content-Type': 'application/x-ndjson',
           Connection: 'keep-alive',
           'Transfer-Encoding': 'chunked',
-          'Cache-Control': 'no-cache',
+          'Cache-Control': 'private, no-cache, no-store, must-revalidate',
```

Comment:
As it's now the default value, can't this line just be removed?

Reply:
Yup, good catch